docs/REST API/rest-api.md

```diff
@@ -10801,6 +10801,7 @@ None.
       "force_password_reset": false,
       "gravatar_url": "",
       "sso_enabled": false,
+      "two_factor_authentication_enabled": false,
```

Comment:
Kind of a long key name but we spell out "authentication" in other places so felt weird to just put "auth"

Reply:
Alternatively, could call this:
```suggestion
      "2fa_enabled": false,
```
But I figured it's clearer if we spell it out.

Reply:
Hey @rachaelshaw does this API design give us room to add different 2FA methods later? 

We're only supporting email verification now but in the future we might support an authenticator app and/or security key. We might want to allow setting different methods for each user.

Reply:
OMG I missed your comment here 😅 https://github.com/fleetdm/fleet/issues/22078#issuecomment-2384179715

The answer is yes

Reply:
@noahtalerman if we definitely think there's a good chance of adding other 2FA methods, it might make sense to design this as an object instead (because I assume with other methods there may be additional configuration involved). 
```suggestion
      "two_factor_authentication": {
        "enabled": false
      }
```
(from there we could add `"type": "…"` and then any other configuration we'd need to save)

Reply:
@rachaelshaw I think up to you.

> if we definitely think there's a good chance of adding other 2FA methods

There's a chance but I don't think it will be something we work on in the next year.

Reply:
Cool, then let's keep it simple for now. We can always add a 2fa object later and deprecate (but still support) `two_factor_authentication_enabled`.